### DISCUSSION TITLE
Welcome to xarray Discussions!

### DISCUSSION THREAD

**Discussion by shoyer** [pydata/xarray#4572] [Show and tell]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by andersy005** [comment]
This is awesome! So glad to see this feature enabled 🚀

**Comment by krishnaap** [comment]
Can I update the xarray to a newer version from conda?

**Reply by andersy005** [reply]
> Can I update the xarray to a newer version from conda?

@krishnaap, The newest/latest release (v0.16.2) hasn't landed in conda repositories yet (https://github.com/conda-forge/xarray-feedstock/pull/59).... It should be there shortly though... So, I'd wait a couple of hours and try again once https://github.com/conda-forge/xarray-feedstock/pull/59 has been merged.... 

P.S. In the future, you may want to create a [separate thread](https://github.com/pydata/xarray/discussions/new) for a new topic so as to avoid derailing discussions in an existing thread. It makes it easy to keep tabs on what issues  have/haven't been addressed :)

**Reply by krishnaap** [reply]
Thanks, @andersy005 .
